TOC/Proposal/voting-rules.md

```diff
@@ -0,0 +1,23 @@
+# Voting Rules for TOC Members
+
+This guideline is used for the decision-making process of TOC members in the TiDB community. Refer to Apache's voting rules, decisions are usually made by [lazy consensus](https://www.apache.org/foundation/glossary.html#LazyConsensus). If no consensus can be reached, the matter may be resolved by a majority vote. And potentially highly-controversial matters are recommended to be resolved by a majority vote, too.
+
+## Voting
+
+For all votes, voting must be open for at least 2 days. The deadline should be clearly stated in the call to vote. Votes are clearly indicated by a pull request adding an entry under the proposal folder.
```

Comment:
> The deadline should be clearly stated in the call to vote.

If it is the case, we should mention this in the TEMPLATE. For example,

```
# A Vote for My Proposal

The vote will be open for at least {duration} unless there is an objection or not enough votes.

## Proposal
...
```

Reply:
You are right, thanks.